fmod-oxide/src/core/dsp/effects.rs

```diff
@@ -648,7 +669,9 @@ pub mod oscillator {
 
     #[derive(Debug, Clone, Copy, PartialEq, Eq)]
     #[derive(num_enum::TryFromPrimitive, num_enum::IntoPrimitive)]
-    #[repr(u32)]
+    // stupid enum repr hack
+    #[cfg_attr(target_env = "msvc", repr(i32))]
+    #[cfg_attr(not(target_env = "msvc"), repr(u32))]
```

Comment:
Not 100% sure this one is correct to use i32 on msvc? Since here we are not referring to FMOD_* constants from bindgen, but rather our own hardcoded values. Thoughts?